src/modules/75-cd/components/PipelineSteps/Common/Terraform/Editview/TerraformArtifactoryForm.tsx

```diff
@@ -193,25 +194,43 @@ export const TFArtifactoryForm: React.FC<StepProps<any> & TFArtifactoryProps> =
                     <FormInput.Text name="varFile.identifier" label={getString('identifier')} />
                   </div>
                 )}
-
+                <Text style={{ color: Color.GREY_900 }}>
+                  {getString('pipelineSteps.repoName')}
+                  <Button
+                    style={{ padding: 0, margin: 0 }}
+                    icon="tooltip-icon"
+                    minimal
+                    color={Color.PRIMARY_7}
+                    tooltip={getString('cd.artifactRepoTooltip')}
+                    tooltipProps={{
+                      usePortal: false,
+                      isDark: true,
+                      className: css.tooltipProps
+                    }}
+                    iconProps={{
+                      size: 12,
+                      padding: { right: 'small', top: 'small', bottom: 'small' }
+                    }}
+                  />
+                </Text>
                 <div className={cx(stepCss.formGroup, stepCss.md)}>
                   {getMultiTypeFromValue(connectorRef) === MultiTypeInputType.FIXED ? (
                     <FormInput.MultiTypeInput
                       selectItems={connectorRepos ? connectorRepos : []}
                       name={tfArtifactoryFormInputNames(isConfig).repositoryName}
-                      label={getString('pipelineSteps.repoName')}
+                      label={''}
```

Comment:
Might be blank, but it's still a string, so you can drop the parens (same below)
```suggestion
                      label=""
```